backend/tests/mw_drw_proxy/proxy.py

```diff
@@ -0,0 +1,165 @@
+
+DRW_URL_PREFIX = os.getenv("DRW_URL_PREFIX", "https://demo-wallet.diem.com")
+
+
+def error_response(e):
+    status_code = 400
+    if isinstance(e, HTTPError):
+        status_code = e.response.status_code
+    return {
+        "error": str(e),
+        "stacktrace": traceback.format_tb(e.__traceback__),
+    }, status_code
+
+
+@app.route("/reset", methods=["GET", "POST"])
+def reset_test():
+    STATE.clear()
+
+
+@app.route("/accounts", methods=["POST"])
+def create_account():
+    try:
+        data = request.json
+
+        # "{\"type\": \"individual\", \"payload_version\": 1, \"given_name\": \"Tom\", \"surname\": \"Jack\"}"
+        kyc_data = json.loads(data["kyc_data"])
+        kyc_info = {
+            "first_name": kyc_data["given_name"],
+            "last_name": kyc_data["surname"],
+            "dob": "1956-07-09",
+            "phone": "1 202-456-1414",
+            "address_1": "1840 Century Park East",
+            "address_2": "",
+            "city": "Los Angeles",
+            "state": "California",
+            "zip": "90067",
+            "country": "US",
+            "selected_fiat_currency": "USD",
+            "selected_language": "en",
+        }
+        uc = UserClient.create(
+            DRW_URL_PREFIX,
+            "_".join([kyc_data["given_name"], kyc_data["surname"]]).lower(),
+            kyc_info=kyc_info,
+            logger=logging.info,
+        )
+
+        for currency, amount in data["balances"].items():
+            if amount > 0:
+                uc.buy(amount, currency, "USD")
+
+        STATE[uc.name] = uc
+
+        return ({"id": uc.name, "kyc_data": data["kyc_data"]}, 200)
+
+    except Exception as e:
+        return error_response(e)
+
+
+@app.route("/accounts/<account_id>/balances", methods=["GET"])
+def account_balances(account_id):
+    try:
+        return {b["currency"]: b["balance"] for b in STATE[account_id].get_balances()}
+
+    except Exception as e:
+        return error_response(e)
+
+
+@app.route("/accounts/<account_id>/send_payment", methods=["POST"])
+def send_payment(account_id):
+    try:
+        # { "currency": "XUS", "amount": 0, "payee": "string" }
+        data: dict = request.json
+        res = STATE[account_id].transfer(
+            addr=data["payee"], amount=data["amount"], currency=data["currency"]
+        )
+
+        return {
+            "id": res["id"],
+            "account_id": account_id,
+            "currency": res["currency"],
+            "amount": res["amount"],
+            "payee": data["payee"],
+        }
+
+    except Exception as e:
+        return error_response(e)
+
+
+@app.route("/accounts/<account_id>/payment_uris", methods=["POST"])
+def get_payment_uri(account_id):
+    try:
+        address = STATE[account_id].get_recv_addr()
+        return {
+            "id": address,
+            "account_id": account_id,
+            "currency": "XUS",
+            "amount": 0,
+            "payment_uri": f"{address}?c=XUS&am=0",
```

Comment:
it is valid to have no c and am params, if it is not provided.
it probably should be "diem://{address}"

better way to handle it is: 
```
from diem import identifier
.....

identifier.encode_intent(address) # when currency and amount is provided, add into params.
```

Reply:
![image](https://user-images.githubusercontent.com/1482859/109727442-35d70a80-7b69-11eb-95e4-b566725ca1ac.png)
from the api spec, it looks like currency/amount are never provided, but looks like it was expected in the output

Reply:
I think this is tool issue, see what I have (you get this by dmw start-server, and visit http://localhost:8888):

<img width="888" alt="Screen Shot 2021-03-02 at 4 45 24 PM" src="https://user-images.githubusercontent.com/17474/109735001-cf58e900-7b76-11eb-844b-08c3c46bdec8.png">